cinema/views.py

```diff
@@ -1 +1,51 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from cinema.serializers import GenreSerializer, ActorSerializer, \
+    CinemaHallSerializer, MovieSerializer, MovieSessionSerializer, \
```

Comment:
Import style looks not so good. 
Try black module. [https://pypi.org/project/black/](url)